### ISSUE TITLE
fix useAdjusted()

### ISSUE THREAD

**Issue by dankelley** [ArgoCanada/argoFloats#260]
I broke this (as seen below) in coding to let '[[' return computed (not stored) values.  I plan to fix this tomorrow morning, and am only making an issue report because I accidentally pushed the code.

Luckily, we have a good test suite that catches such errors!

```
> testthat::test_check("argoFloats")
── 1. Failure: useAdjusted() test (@test_adjusted.R#20)  ───────────────────────────────────────────────────────
a[["salinity"]] not equal to a[["PSAL_ADJUSTED"]].
Component 1: Mean relative difference: 3.52753e-05

── 2. Failure: useAdjusted() test (@test_adjusted.R#24)  ───────────────────────────────────────────────────────
r[[paste0(field, "Adjusted")]] not equal to a[[field]].
Component 1: Mean relative difference: 3.527405e-05

── 3. Failure: useAdjusted() test (@test_adjusted.R#20)  ───────────────────────────────────────────────────────
a[["salinity"]] not equal to a[["PSAL_ADJUSTED"]].
Component 1: Mean relative difference: 6.854856e-07

── 4. Failure: useAdjusted() test (@test_adjusted.R#22)  ───────────────────────────────────────────────────────
a[["pressure"]] not equal to a[["PRES_ADJUSTED"]].
Component 1: Mean relative difference: 6.416215e-05

── 5. Failure: useAdjusted() test (@test_adjusted.R#24)  ───────────────────────────────────────────────────────
r[[paste0(field, "Adjusted")]] not equal to a[[field]].
Component 1: Mean relative difference: 6.854854e-07

── 6. Failure: useAdjusted() test (@test_adjusted.R#24)  ───────────────────────────────────────────────────────
r[[paste0(field, "Adjusted")]] not equal to a[[field]].
Component 1: Mean relative difference: 6.415803e-05

══ testthat results  ═══════════════════════════════════════════════════════════════════════════════════════════
[ OK: 226 | SKIPPED: 1 | WARNINGS: 0 | FAILED: 6 ]
1. Failure: useAdjusted() test (@test_adjusted.R#20) 
2. Failure: useAdjusted() test (@test_adjusted.R#24) 
3. Failure: useAdjusted() test (@test_adjusted.R#20) 
4. Failure: useAdjusted() test (@test_adjusted.R#22) 
5. Failure: useAdjusted() test (@test_adjusted.R#24) 
6. Failure: useAdjusted() test (@test_adjusted.R#24) 
```

**Comment by dankelley** [comment]
This and #261 are both fixed in argoFloats "develop" commit 6122306ad4492473de7a2de6cddb34d1c082607f with oce "develop" commit 53ac1bbd53d434e4844517b12bdee419386c5323.

I am not entirely sure what the problem was, to be frank.  I had convinced myself that we could not use `oce::[[,argo-method` because the S4-"signature" scheme did not seem to be catching it.  So, yesterday, I wrote code to grab data, make a ctd object, etc., with still no luck.  Fiddling with it today, I tried again just using `oce::[[,argo-method` and it worked.  So I suspect this was a messup on my part somehow.  Maybe the S4-"signature" scheme was mixed up by something I had been doing yesterday (in RStudio ... always risky because just what does it actually clean when you click on the cleanup icon ...??).

Long story short, I ended up commenting out 15 or so lines, keeping them just in case, but now the method is very simple: I use an `lapply()` within which I use a `[[`.

Since the test suite builds, I am closing this and in a second I'll also close the related #261.
